qubesadmin/tools/qvm_template.py

```diff
@@ -0,0 +1,1396 @@
+#!/usr/bin/env python3
+
+'''Tool for managing VM templates.'''
+
+import argparse
+import collections
+import datetime
+import enum
+import fnmatch
+import functools
+import itertools
+import json
+import operator
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+import typing
+
+import qubesadmin
+import qubesadmin.tools
+import qubesadmin.tools.qvm_kill
+import qubesadmin.tools.qvm_remove
+import rpm
+import tqdm
+import xdg.BaseDirectory
+
+PATH_PREFIX = '/var/lib/qubes/vm-templates'
+TEMP_DIR = '/var/tmp'
+PACKAGE_NAME_PREFIX = 'qubes-template-'
+CACHE_DIR = os.path.join(xdg.BaseDirectory.xdg_cache_home, 'qvm-template')
+UNVERIFIED_SUFFIX = '.unverified'
+LOCK_FILE = '/var/tmp/qvm-template.lck'
+
+def qubes_release() -> str:
+    """Return the Qubes release."""
+    if os.path.exists('/usr/share/qubes/marker-vm'):
+        with open('/usr/share/qubes/marker-vm', 'r') as fd:
+            # Get last line (in the format `x.x`)
+            return fd.readlines()[-1].strip()
+    return subprocess.check_output(['lsb_release', '-sr'],
```

Comment:
Either this is missing a dependency (there's no lsb_release in dom0 by default), or maybe you could just directly read /etc/os-release ? it seems a bit overkill to use a separate process and program to parse a simple file - especially as redhat-lsb-core has several otherwise useless dependencies.

Reply:
Replaced with reading /etc/os-release.

(Did not realize that there's no lsb_release by default -- upon inspection it seemed to be pulled in by `tlp` on my box.)